js/plot/message.js

```diff
@@ -129,6 +131,58 @@ module.exports = function(pool, opts) {
     });
   };
 
+  /**
+   * Render the affordance for adding notes through blip
+   */
+  message.drawNewNoteIcon = _.once(function() {
+    if (!d3.select('#tidelineLabels .newNoteIcon').empty()) { // do not draw twice!
+      return;
+    }
+
+    var newNote = d3.select('#tidelineLabels').append('image')
+      .attr({
+        'class': 'newNoteIcon',
+        'xlink:href': newNoteImg,
+        cursor: 'pointer',
+        x: 0,
+        y: 45,
+        width: 36,
+        height: 29
+      });
+
+    newNote.on('mouseover', function() {
+      d3.select('#tidelineLabels').append('text')
+      .attr({
+        'class': 'newNoteText',
+        x: 1,
+        y: 88,
```

Comment:
Oi, anything we can do about these magic numbers? Are they related to the size of the image asset?